client/src/routes/Session/state/sessionNotificationsState.ts

```diff
@@ -0,0 +1,24 @@
+import create from 'zustand';
+import {NotificationProps} from '../components/Notifications/Notification';
+
+type State = {
+  notifications: NotificationProps[];
```

Comment:
Nit pick: Like this reused type, but would maybe do it in reverse. Basically define what a notification is in this file, and then use that definition for the props in the component :)

Reply:
100%